### PR TITLE
feat(TV UI): Player's Top controls redesign

### PR DESCRIPTION
**Pull request by KingLucius**
- Fixes Title alignment in case of RTL languages.
- Adds "Restart" & "Next Episode" in top controls bar.
- Marquee under button titles.
- Removes the "Next Episode" button under progress seekbar.
- Changes "Tracks" icon.
- Margin for top bar is same on right & left.
- Removes the back button from player loading UI.
- Dialog for reset subtitles delay confirmation.
- Make reset subtitles + & - button focus smaller.

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/ui/player/FullScreenPlayer.kt [hunk 1]
```diff
@@ -493,9 +494,17 @@ open class FullScreenPlayer : AbstractPlayerFragment() {
                 player.seekTime(1L)
             }
             resetBtt.setOnClickListener {
-                subtitleDelay = 0
-                dialog.dismissSafe(activity)
-                player.seekTime(1L)
+                val resetSubsBuilder: AlertDialog.Builder = AlertDialog.Builder(ctx)
+                resetSubsBuilder.setTitle(R.string.confirm_reset_subs_dialog)
+                resetSubsBuilder.apply {
```
Comment (fire-light42) [inline app/src/main/java/com/lagradost/cloudstream3/ui/player/FullScreenPlayer.kt:499]: I am not a big fan of confirmation button for such a minor thing.

**Review by fire-light42 — COMMENTED** [review]
Very nice code and UI 👍

File: app/src/main/java/com/lagradost/cloudstream3/ui/player/FullScreenPlayer.kt (hunk 1 above)
Comment (KingLucius) [reply app/src/main/java/com/lagradost/cloudstream3/ui/player/FullScreenPlayer.kt:499]: I thought you asked for that change in other PR, I think I got you wrong
Will revert it

Comment (fire-light42) [reply app/src/main/java/com/lagradost/cloudstream3/ui/player/FullScreenPlayer.kt]: I meant for it on the `Reset video` button you added, and not `Reset subtitles`, but that was not explain that properly.

**Review by fire-light42 — APPROVED** [review]

File: app/src/main/java/com/lagradost/cloudstream3/ui/player/FullScreenPlayer.kt (hunk 1 above)
Comment (KingLucius) [reply app/src/main/java/com/lagradost/cloudstream3/ui/player/FullScreenPlayer.kt]: Oh, you think we will need this really?

**Comment by KingLucius** [comment]
> Anything else, or do you want this merged?

If you are OK with restart video button without confirmation so go ahead 👍

**Pull request merged**